api/src/integrations/epic/epic-utils.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Additional common Epic pre-processing/transformations for Epic records.
+ *
+ * @param {object} epicRecord Epic record (required)
+ * @returns {Order} updated Epic record.
+ * @throws {Error} if record is not provided.
+ */
+const preTransformRecord = function(epicRecord) {
+  if (!epicRecord) {
+    throw Error('preTransformRecord - required record must be non-null.');
+  }
+
+  // Transform Epic project names to match values tracked by NRPTI
+  if (epicRecord.project && epicRecord.project.name) {
+    switch (epicRecord.project.name) {
+      case 'LNG Canada Export Terminal':
+        epicRecord.project.name = 'LNG Canada';
+        break;
+      case 'Coastal GasLink Pipeline':
+        epicRecord.project.name = 'Coastal Gaslink';
+    }
+  }
+
+  // Transform Epic author names to match values tracked by NRPTI
+  if (epicRecord.documentAuthor) {
+    switch (epicRecord.documentAuthor) {
```

Comment:
Are these documentAuthor fields valid against the EPIC dataset?  I was under the impression that documentAuthor fields in EPIC are the actual names of people, or null, or empty string.  I can't validate this as I no longer have read access to EPIC DB.

Reply:
Ill be honest, this was a random update from Lena, that said to map epic author to a canned nrpti author value.  documentAuthor is the only value on the record, so I just assumed that was the one she meant.

Looking at the response though, it looks like documentAuthor is generally null, or an _id, so Ill have to ask her what she was thinking around this more in depth.